kong/plugins/prometheus/exporter.lua

```diff
@@ -425,7 +476,7 @@ local function metric_data(write_fn)
     end
   end
 
-  prometheus:metric_data(write_fn)
+  prometheus:metric_data(write_fn, not is_prometheus_enabled())
```

Comment:
Better add a comment here to explain why `not is_prometheus_enabled()` is used. I had to dig into `prometheus.lua` to find out.